index.js

```diff
@@ -7,11 +7,27 @@
  * @end
  * -----------------------------------------------------------------------------
  */
+var exports = module.exports;
 
-module.exports = {
-    Updater: require('./updater')
-    Master: require('./master'),
-    Collector: require('./collector'),
-    Scaler: require('./scaler')
-};
+/**
+ * Master exports
+ * 
+ */
+var master = require('./master');
+exports.AlAzureMaster = master.AlAzureMaster;
```

Comment:
Not just require('./master').AlAzureMaster (and then can use the previous syntax do export everything in a single object def)?

Reply:
I did it this way in case there will be more function/classes/constants exported.